osquery/tables/system/windows/userassist.cpp

```diff
@@ -93,7 +31,6 @@ std::size_t executionNum(const std::string& assist_data) {
   }
 
   std::string execution_count = assist_data.substr(8, 8);
-
```

Comment:
Try not to include "spurious" changes like this in larger PRs. Small code clean ups such as removing newlines should be done in dedicates PRs.

Reply:
understood
I did make few more changes in this PR (added a test for filetime conversion), but going forward will open new PRs